react/intl-polyfill.ts

```diff
@@ -0,0 +1,33 @@
+import { path } from 'ramda'
+
+const locale = path<string>(['__RUNTIME__', 'culture', 'locale'], window)
+const [lang] = locale ? locale.split('-') : ''
+
+if (lang && window.Intl && !window.Intl.PluralRules) {
+  if (!window.IntlPluralRules || !window.IntlPluralRules.PluralRules) {
+    import('@formatjs/intl-pluralrules/polyfill' as any).then(() => {
```

Comment:
I think we could always import both polyfills. We are creating a waterfall of dynamic imports that probably will give us a worse page load with slow connections.